### ISSUE TITLE
doctest sbt command no longer fails when individual test fails

### ISSUE THREAD

**Issue by kailuowang** [typelevel/cats#2758]
This is quite puzzling to me. 
When I run `sbt coreJVM/test` after intentionally breaking a doc test. 
I can see
```
[info] ! NonEmptyChain.scala.NonEmptyChain.scala:63: fromSeq.example at line 67: cats.data.NonEmptyChain.fromSeq(collection.immutable.Seq(12, ...: Falsified after 0 passed tests.
[info] > Labels of failing property: 
[info] 'Some(Chain(12, 2, 3))' is not equal to 'Some(Chain(1, 2, 3))'
```
but the final result is 
```
[info] Passed: Total 66, Failed 0, Errors 0, Passed 66
[success] Total time: 12 s, completed Mar 25, 2019 9:31:48 PM
```

One thing I noticed that is the total test `66` seems very low.

**Comment by ceedubs** [comment]
@kailuowang I can't explain it, but I think that #2734 fixes this issue. See my comment in #2735.

**Comment by kailuowang** [comment]
ah I misread your comment there. thanks for pointing that out. In that case I propose we merge #2734 now just to fix this (it's scary to have failing test passing build)

**Comment by kailuowang** [comment]
Confirmed that this is closed by #2734
